packages/web-components/fast-foundation/src/dialog/dialog.template.ts

```diff
@@ -28,7 +28,7 @@ export const dialogTemplate: FoundationElementTemplate<ViewTemplate<Dialog>> = (
             tabindex="-1"
             class="control"
             part="control"
-            aria-modal="${x => x.modal}"
+            aria-modal="${x => (x.modal ? x.modal : void 0)}"
```

Comment:
Should this be a boolean attr binding instead?

Reply:
I don't think so - `modal` itself is a boolean attribute and would leverage that binding in templates, but we're passing the value of that to the `aria-modal` attribute when it's present and not applying that attribute when it's not.

When using the boolean attribute binding here, we're going to end up with `aria-modal` being set on the internal dialog DOM node, but no value, which would not be valid. What we want to end up with is `aria-modal="true"` when `modal` is true, and not setting `aria-modal` at all when it's false.

@eisenbergeffect as FYI since you mentioned as well.

Reply:
Makes sense. All good then.